system/src/system.cpp

```diff
@@ -69,34 +70,41 @@ int lua_threadcount(lua_State* L)
     return 1;
 }
 
+#define BYTES_PER_MB 1048576 // 2^20 bytes
```

Comment:
This shouldn't be a macro, but rather just a constant.

Reply:
```suggestion
constexpr std::size_t BYTES_PER_MB = 1024 * 1024;
```